src/socket/server.js

```diff
@@ -29,25 +29,34 @@ const sendInitialState = (queueId, callback) => {
     callback({ questions, activeStaff })
   })
 }
-
+// const blankQuestion = {
```

Comment:
Would this be needed in the future?

Reply:
Nope, my bad - looks like I didn't delete some old code